src/util.js

```diff
@@ -12,4 +12,13 @@ const humanizeDueTimeForForm = (dueDate) => dueDate ? dayjs(dueDate).format(FORM
 
 const updateItem = (items, update) => items.map((item) => item.id === update.id ? update : item);
 // const updateItem = (item, prop) => ({...item, ...prop});
-export {getRandomArrayElement,humanizeDueDate,humanizeDueTime, machineDueFormat,humanizeDueTimeForForm,updateItem};
+const getTimeDifference = ({dateFrom,dateTo}) => (new Date(dateTo)).getTime() - (new Date(dateFrom)).getTime();
+// isAfter
+const sortEventsBy = {
+  [SortType.DAY]: (events) => [...events],
+  [SortType.TIME]: (events) => [...events].sort((a,b) => getTimeDifference(b) - getTimeDifference(a)),
+  [SortType.PRICE]: (events) => [...events].sort((a,b) => b.basePrice - a.basePrice),
```

Comment:
Такое именование аргументов будет нарушением критерия [Б4](https://up.htmlacademy.ru/ecmascript/23/criteries#basic)

```suggestion
  [SortType.PRICE]: (events) => [...events].sort((nextEvent, currentEvent) => currentEvent.basePrice - nextEvent.basePrice),
```